client/aws/errors.go

```diff
@@ -1,5 +1,7 @@
 package aws
 
 const (
-	KeyPairDuplicate = "InvalidKeyPair.Duplicate"
+	BucketAlreadyExists     = "BucketAlreadyExists"
+	BucketAlreadyOwnedByYou = "BucketAlreadyOwnedByYou"
```

Comment:
`Ỳou` is the user in AWS/S3. This error occurs if the bucket already exists and was created by your AWS user.